### ISSUE TITLE
adding "pp-scrollable" is not working

### ISSUE THREAD

**Issue by welbinator** [alvarotrigo/pagePiling.js#189]
It breaks some of my CSS and doesn't add a scroll bar

https://welbinator.github.io/crwd/#services

https://github.com/welbinator/crwd

**Comment by welbinator** [comment]
Ok I was a bit mistaken.

So it does add a scrollbar. And the reason it was "Messing up" my css is because apparently every other section gets a div called pp-tablecell but if you add pp-scrollable, then it doesn't get that extra div. So I fixed that bit.

But for some reason, on mobile, it's pushing all my content up and you can't see it. 

I mean I guess I can just give it like a huge top margin but that seems hacky. Is there something I can do to fix this issue where all the content on a pp-scrollable seciton gets pushed up super far?

**Comment by alvarotrigo** [comment]
I don't see any issue.

**Comment by welbinator** [comment]
yeah I fixed the CSS manually.

For some reason, I had an overlay div, pretty simple width:height 100% background black opacity .5 and I was using object-fit and gave it position:fixed but with pp-scrollable enabled, that overlay div just wouldn't sit still. It would scroll with the rest of the content.

So I took out the overlay div and just added a black overlay to the image itself.

Then I had to add some margin-top to the top most element to make all the content come down because when pp-scrollable is enabled, for some reason it pushes all the content up.
